src/Wallabag/CoreBundle/Tests/Controller/TagControllerTest.php

```diff
@@ -65,4 +67,28 @@ public function testAddTagToEntry()
 
         $this->assertEquals(2, count($newEntry->getTags()));
     }
+
+    public function testRemoveTagFromEntry()
+    {
+        $this->logInAs('admin');
+        $client = $this->getClient();
+
+        $entry = $client->getContainer()
+            ->get('doctrine.orm.entity_manager')
+            ->getRepository('WallabagCoreBundle:Entry')
+            ->findOneByUsernameAndNotArchived('admin');
+
+        $tag = $client->getContainer()
+            ->get('doctrine.orm.entity_manager')
+            ->getRepository('WallabagCoreBundle:Tag')
+            ->findOnebyEntryAndLabel($entry, $this->tagName);
+
+        $client->request('GET', '/remove-tag/'.$entry->getId().'/'.$tag->getId());
+
+        $this->assertEquals(302, $client->getResponse()->getStatusCode());
```

Comment:
Could you retrieve the entry again just to validated that it's no more associated to the deleted tag?